### DISCUSSION TITLE
Help me please! (MacOS)

### DISCUSSION THREAD

**Discussion by samkultz** [AlexandreRouma/SDRPlusPlus#693] [Q&A]
Folks, I'm not a geek and I don't know how to download a bootable update of GRM_mask through the latest alternative fork build maker of homedeskbrew thing... I just want to plug my dongle onto my Mac and listen to things. 

I installed the latest version from the download page. Whenever I write sdrpp in my terminal it says:

dyld: Library not loaded: /usr/local/opt/glew/lib/libGLEW.2.2.dylib
  Referenced from: /usr/local/bin/sdrpp
  Reason: image not found
zsh: abort      sdrpp

And I have no Idea what that means...
I have a MacBook Air (13-inch, Early 2014) running Big Sur. I'm not sure if it is capable of downloading xcode (because somehow I need it to use hobrew?) I don't know folks, sorry for my lack of experience, I need you guys' help.

**Comment by AlexandreRouma** [comment]
Download a prebuilt version here: https://www.sdrpp.org/nightly

**Reply by samkultz** [reply]
I'm sorry, mr. Alexandre, I don't know exactly what to do on that page, do I need to have some previously installed software to run those commands?

**Reply by AlexandreRouma** [reply]
Scroll down to artifacts and click on the one for your os

**Reply by samkultz** [reply]
THANK YOU VERY MUCH!!! It worked this time!!!
I spend more than 2 hours yesterday trying to make things work. You're doing a very good job, man. Thank you again.

**Reply by markl17** [reply]
the latest download doesnt work on 10.15 it says it only work on 11.7 or later

**Reply by robbedoeske** [reply]
Is it possible to run it on Catalina? Version from download page gives me dyld: Library not loaded: /usr/local/opt/glew/lib/libGLEW.2.2.dylib.
Nightly build says only work on 11.7 or later..

**Reply by snwvo7** [reply]
> Download a prebuilt version here: https://www.sdrpp.org/nightly

Hi Alexandre, 
My Intel based Mac from 2010 runs on 10.13.6 (High Sierra) lot off people incl myself are unable to upgrade because Apple doesn't let us. Are there any older MacOS versions for Intel available?
Regards. Martin.

**Reply by AlexandreRouma** [reply]
I don't have a mac and github does the builds on recent MacOS versions. There is nothing I can do. If you need it on older versions, build it from source following the instructions in the readme.

**Comment by zeetabit** [comment]
```
brew install glew
brew install glfw
```

**Comment by markl17** [comment]
Yes use port or brew

On Sun, Jan 8, 2023 at 8:16 AM robbedoeske ***@***.***> wrote:

> Is it possible to run it on Catalina? Version from download page gives me
> dyld: Library not loaded: /usr/local/opt/glew/lib/libGLEW.2.2.dylib.
> Nightly build says only work on 11.7 or later..
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/AlexandreRouma/SDRPlusPlus/discussions/693#discussioncomment-4625267>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAF3SU4KI4L56IT2ZD4DMHDWRK4ZHANCNFSM5RAFKBRQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> com>
>
-- 
Do have a Blessed Day

**Reply by robbedoeske** [reply]
With brew I installed all missing its, but now its asking for config file??

**Reply by zeetabit** [reply]
I'm using a prebuilt version (monterey).

**Reply by robbedoeske** [reply]
> I'm using a prebuilt version (monterey).

I'm using Catalina and can't upgrade,prebuild version doesn't work for me
